src/backend/common/CommandBuffer.cpp

```diff
@@ -472,6 +594,28 @@ namespace backend {
                             return false;
                         }
 
+                        // TODO(kainino@chromium.org): We should be able to
+                        // optimize this by tracking which textures are in use
+                        // as attachments. Maybe it's possible that the
+                        // XAttachment usages could mark this: disallow
```

Comment:
(Or leaving their attachments in the 'XAttachment' usage, because it doesn't matter whether it's that or 'no usage', I think.)